scripts/combine/pullsAndImpacts.py

```diff
@@ -303,6 +294,7 @@ def parseArgs():
     parser.add_argument("--debug", action='store_true', help="Print debug output")
     parser.add_argument("--oneSidedImpacts", action='store_true', help="Make impacts one-sided")
     parser.add_argument("--filters", nargs="*", type=str, help="Filter regexes to select nuisances by name")
```

Comment:
We probably want a similar feature for the filters where they can be read from a file. Then we can have default files that either give a maximally grouped subset or a maximally split one. 

The mix of the two is confusing. To this end it might be worth adopting a naming scheme so we can tell the difference between a split vs. grouped syst, which would make it more clear what to look at if you look at everything together and easier to filter.

Reply:
I like the suggestion to make a file with the maximally grouped/subgrouped cases and can add this. Not sure if necessary to keep track in a naming scheme, I use the splitGroup to make sub and super groupings. If we have the files one can see quickly what are sub and super groups